server/consumer.go

```diff
@@ -295,7 +295,7 @@ const (
 	// outstanding.
 	JsFlowControlMaxPending = 1 * 1024 * 1024
 	// JsDefaultMaxAckPending is set for consumers with explicit ack that do not set the max ack pending.
-	JsDefaultMaxAckPending = 20_000
+	JsDefaultMaxAckPending = 1000
```

Comment:
I decided to use 1,000 instead of 1,024 as we tend to do for any kind of limit, even when not bytes limit (well, this is a bad example since previous value was 20,000, not 20*1024). Let me know if you agree with this approach.